src/foam/dao/ArrayDAO.js

```diff
@@ -29,6 +29,14 @@ foam.CLASS({
 
   properties: [
     {
+      class: 'Class',
+      name: 'of',
+      factory: function() {
+        if (this.array.length === 0) return foam.lookup('foam.core.FObject');
```

Comment:
this.lookup is fine without an import currently.

I think Kevin was implying that you could just reference foam.core.FObject globally directly, since it will always be defined.  But a lookup doesn't really hurt I don't think.